website/docs/r/synthetics_script_monitor.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_synthetics_script_monitor"
+sidebar_current: "docs-newrelic-resource-synthetics-script-monitor"
+description: |-
+Create and manage a Synthetics script monitor in New Relic.
+---
+
+# Resource: newrelic\_synthetics\_script\_monitor
+
+Use this resource to create and manage New Relic synthetics script monitor.
```

Comment:
Use this resource to create, update, and delete a Script API or Script Browser Synthetics Monitor in New Relic.

Reply:
Done